docs/Hooks.md

```diff
@@ -0,0 +1,27 @@
+# CIS Notifications
+
+Change Integration Service version 2 features outbound hooks to notify relying parties that the user
+profile may have changed.
+
+## Basic Flow 
+
+The basic flow is as follows.
+
+1. Publisher triggers an authenticated update.
+2. It is written to the Dynamodb table using dynamodb transactions. 
+3. An event is generated to a dynamodb stream.
+4. A webhook service iterates through the single sharded/single partition stream in batches of 10 users.
+5. A request payload is generated that contains the following information.
+```
+    {
+        "operation": update,
+        "id": ad|Mozilla-LDAP|dinomcvouch,
```

Comment:
should we add `uuid`?

Reply:
We can't.  We only get the PK in the event.  I could unwrap these to full profiles but no one is asking for UUID.